installer_scripts/install-yb-voyager-debezium

```diff
@@ -0,0 +1,229 @@
+main() {
+	set -x
+
+	os=$(uname -s)
+	case ${os} in
+		"Darwin")
+			install_core_utils_macos
+			exit $?
+			;;
+		"Linux")
+			# Proceed to distribution check.
+			;;
+		*)
+			echo "ERROR: unsupported os ${os}"
+			exit 1
+			;;
+	esac
+
+	# Linux
+	dist=$(cat /etc/os-release | grep -w NAME | awk -F'=' '{print $2}' | tr -d '"')
+	case ${dist} in
+		"CentOS Linux")
+			install_core_utils_centos
+			;;
+		"AlmaLinux")
+			install_core_utils_centos
+			;;
+		"Red Hat Enterprise Linux")
+			install_core_utils_centos
+			;;
+		"Red Hat Enterprise Linux Server")
+			install_core_utils_centos
+			;;
+		"Ubuntu")
+			install_core_utils_ubuntu
+			;;
+		*)
+			echo "ERROR: unsupported linux distribution: ${dist}"
+			exit 1
+			;;
+	esac
+
+	check_java
+	check_install_maven
+	install_debezium_server
+	output "Installation complete."
+}
+
+#=============================================================================
+# COMMON
+#=============================================================================
+
+check_binutils_version() {
+	output "Checking binutils version."
+	min_required_version='2.25'
+
+	# Example output of "ld -v" on CentOS/RHEL:
+	# GNU ld version 2.30-113.el8
+	# Example output of "ld -v" on Ubuntu:
+	# GNU ld (GNU Binutils for Ubuntu) 2.38
+	version=$(ld -v | awk '{print $NF}' | awk -F '-' '{print $1}')
+
+	if [[ ! "$version" > "$min_required_version" ]] 
+	then
+		echo "ERROR: unsupported binutils version ${version}. Update to binutils version > ${min_required_version} ."
+		exit 1
+	fi
+}
+
+centos_check_base_repo_enabled() {
+
+	output "Checking if base repo is enabled."
+
+	if [[ -z "$(yum repolist enabled | grep base)" ]]
+	then 
+		echo "ERROR: base repo is not enabled. Enable the repo and retry."
+		exit 1
+	fi
+}
+
+#=============================================================================
+# CENTOS
+#=============================================================================
+
+install_core_utils_centos() {
+	check_binutils_version
+	centos_check_base_repo_enabled
+	$YUM which wget git gcc make 1>&2
+}
+
+#=============================================================================
+# UBUNTU
+#=============================================================================
+
+install_core_utils_ubuntu() {
+	check_binutils_version
+	sudo apt update 1>&2
+	sudo apt-get -y install wget make 1>&2
+}
+
+#=============================================================================
+# MacOS
+#=============================================================================
+
+install_core_utils_macos() {
+	macos_install_brew
+	brew install wget
+}
+
+macos_install_brew() {
+	which brew > /dev/null 2>&1
+	if [ $? -eq 0 ]
+	then
+		output "brew is already installed."
+	else
+		output "Installing brew."
+		set +x  # Do not dump the brew install.sh into the log.
+		/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
Will it send all the contents of install.sh on the command line of bash?

I would suggest to download the `install.sh` and then pass the name of the file to bash.

Reply:
Yes, it will download the contents of install.sh and pass it on to bash -c. It's the official installation command provided by homebrew. 
What concern do you see with this? In fact, I think it's better because skips the need to delete the installation script after it's done.